Classes/New Issue/IssueTemplates.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  IssueTemplates.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/3/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+struct IssueTemplate {
+    let title:String
+    let template:String
+}
+
+enum IssueTemplateHelper {
+
+    static private func matchesForRegexInText(
+        regex: String!,
```

Comment:
What about `matches(forRegex:inText:)`?
And why are string types force unwrapped?